torchvision/prototype/features/_image.py

```diff
@@ -77,7 +78,9 @@ def guess_color_space(data: torch.Tensor) -> ColorSpace:
             return ColorSpace.OTHER
 
     def show(self) -> None:
+        # This is a nice to have, but not a necessary method, for this early in the prototype
         to_pil_image(make_grid(self.view(-1, *self.shape[-3:]))).show()
 
     def draw_bounding_box(self, bounding_box: BoundingBox, **kwargs: Any) -> "Image":
+        # Same as above and nothing that this is the only method that requires to_format().
```

Comment:
Even without the method call, this is really helpful while debugging a dataset implementation to see at a glance if the bounding boxes are properly set up.